content/articles/how-to-make-a-discord-bot-using-python/index.md

```diff
@@ -0,0 +1,267 @@
+4.	a cloud computing platform called Repl.it.
+
+#### Table of Contents
+
+1. Introduction
+    1. About Discord 
+    2. About Bot
+2.  Overview and Requirement
+3. Creating a Discord Bot Account on the Discord  
+    1.  Forming an Account
+    2. Forming an Application
+    3. Forming a Bot
+    4. Forming a Guild ( a Sever)
+4. Creating a discord bot using discord.py library
+    1. Installing discord.py and create a Repl
+    2. Setting up Discord event for your Bot 
+    3. How to run Bot
+
+
+We will begin with understanding about Discord is and why it’s valuable.
+
+### ABOUT DISCORD
+If you're unfamiliar with Discord, it's a VoIP and chat programme that's meant to take the role of Skype for gamers and educational use. It helps the society to communicate better.
+ 
+ 
+ Discord is used by players, educational institutions, broadcasters, and developers to talk about games, answer queries, communicate while playing and studying, and even more. There's even a gaming store with critical reviews and a streaming service. For gaming communities, it's almost a one-stop store.
+
+
+
+### ABOUT BOT
+Discord is becoming extremely prevalent. As a result, for a community to survive and develop, automatic procedures such as banning unsuitable members and responding to user requests are critical.
+
+Bot users are automated programmes that appear and behave like people and respond to events and orders on Discord automatically. Users of Discord bots (or just bots) have virtually limitless options.
+
+Let's assume you're in charge of a new Discord guild and a new member joins for the first time. If you're feeling very enthusiastic, you may personally contact out to that individual and welcome them to your community. You may also inform them about your channels or request that they introduce themselves.
+
+When individual feels welcomed and likes the conversations in your guild, therefore they invite their friends.
+
+The guild's membership expands to the point that it's no longer practical to individually reach around to each new member, but you always want to send them something that will acknowledge their arrival.
+
+It's possible to automatically respond to a new member entering your guild using a bot. You may even manage how it interacts with each new user by customizing its behavior based on circumstances.
+
+This is fantastic, but it's only a example of how a bot may be beneficial. Once you learn how to build bots, there are a number of ways to get creative with them.
+
+**Note: Even though Discord permits you to develop bots for vocal communication, this tutorial will focus on the texting side of the platform.**
+
+When building a bot, there are two important stages to follow:
+1. Create a Discord bot user and assign it to a guild.
+2. Create code that executes your bot's actions and makes use of Discord's APIs.
+In the following part, you'll discover how to use Discord's Developer Portal to create a Discord bot.
+
+### OVERVIEW AND REQUIREMENTS
```

Comment:
Change all the titles. Only the first letter of the first word is capitalized. 

### Overview and requirements

Reply:
Done